test/visual-diff/visual-tests.json

```diff
@@ -107,6 +107,10 @@
         ".comic-amp-font-loaded",
         ".comic-amp-bold-font-loaded"
       ]
+    },
+    {
+      "url": "examples/visual-tests/amp-layout/amp-layout.amp.html",
+      "name": "AMP Layout"
```

Comment:
I assume there are no css elements to wait for, or special experiments to enable here? (If so, you can do so by adding a `loading-complete-css` or `experiments` section here, as seen in the example at the top of this file.)